.github/workflows/main-checks.yml

```diff
@@ -67,43 +67,67 @@ jobs:
           cargo clippy --release --all-features --all-targets -- --deny=warnings
         working-directory: packages/yew
 
+  spell_check:
+    name: spellcheck
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Check spelling
+        run: |
+          sudo apt-get install aspell
+          ci/spellcheck.sh list
 
   doc_tests:
     name: Documentation Tests
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+        with:
+          path: "./wasm-bindgen"
+          ref: respect-rustdoc-tmp-paths
+          repository: hamza1311/wasm-bindgen
+
+      - uses: Swatinem/rust-cache@v1
+
       - uses: actions-rs/toolchain@v1
         with:
           toolchain: stable
-          target: wasm32-unknown-unknown
           override: true
           profile: minimal
 
-      - uses: Swatinem/rust-cache@v1
+      - run: |
+          cd wasm-bindgen
+          cargo build -p wasm-bindgen-cli --bin wasm-bindgen-test-runner --release
+          cp target/release/wasm-bindgen-test-runner ~/.cargo/bin/wasm-bindgen-test-runner
```

Comment:
This is the best solution I have until https://github.com/rustwasm/wasm-bindgen/pull/2881 is merged and published. I tried building wasm-bindgen-test-runner on my local computer and putting it in `ci` directory but it won't run because it's not built with musl and I can't get a musl build to work locally